drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/ScenarioBeanUtilTest.java

```diff
@@ -90,21 +90,29 @@ public void navigateToObjectTest() {
         dispute.setCreator(creator);
         List<String> pathToProperty = Arrays.asList("creator", "firstName");
 
-        Object targetObject = ScenarioBeanUtil.navigateToObject(dispute, pathToProperty);
+        Object targetObject = ScenarioBeanUtil.navigateToObject(dispute, pathToProperty, true);
 
         assertEquals(targetObject, FIRST_NAME);
     }
 
     @Test(expected = ScenarioException.class)
     public void navigateToObjectNoStepTest() {
-        ScenarioBeanUtil.navigateToObject(new Dispute(), new ArrayList<>());
+        ScenarioBeanUtil.navigateToObject(new Dispute(), new ArrayList<>(), true);
     }
 
     @Test(expected = ScenarioException.class)
     public void navigateToObjectFakeFieldTest() {
         Dispute dispute = new Dispute();
         List<String> pathToProperty = Arrays.asList("fakeField");
 
-        ScenarioBeanUtil.navigateToObject(dispute, pathToProperty);
+        ScenarioBeanUtil.navigateToObject(dispute, pathToProperty, true);
+    }
+
+    @Test(expected = ScenarioException.class)
+    public void navigateToObjectNoStepCreationTest() {
+        Dispute dispute = new Dispute();
+        List<String> pathToProperty = Arrays.asList("creator", "firstName");
+
+        ScenarioBeanUtil.navigateToObject(dispute, pathToProperty, false);
```

Comment:
I would propose again use `assertj` and check also error message.

Reply:
Done